src/openrct2/scripting/ScRide.hpp

```diff
@@ -0,0 +1,154 @@
+/*****************************************************************************
+ * Copyright (c) 2020 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#ifdef __ENABLE_SCRIPTING__
+
+#    include "../common.h"
+#    include "../ride/Ride.h"
+#    include "Duktape.hpp"
+#    include "ScObject.hpp"
+#    include "ScriptEngine.h"
+
+namespace OpenRCT2::Scripting
+{
+    class ScRide
```

Comment:
Class is using snake_cast instead of CamelCast.

Should be `ScriptRide` (file as well)